### PR TITLE
RLM-1425 Set logging to be more verbose by default

### PR DESCRIPTION
**Pull request by rexredinger**
Title says it all.

Somewhat related to discussion here:

Issue: [RLM-1425](https://rpc-openstack.atlassian.net/browse/RLM-1425)

### CODE REVIEW

**Comment by rexredinger** [comment]
recheck_issue_link

**Comment by rexredinger** [comment]
recheck_issue_link

**Comment by hughsaunders** [comment]
Many of the scripts that call this supply `--debug` so this won't have too much effect. 

However I think this is a better approach, we should probably remove all the invocations with `--debug` and set level at the top of each script.

File: scripts/release.py [hunk 1]
```diff
@@ -10,7 +10,8 @@
 import ghutils
 from notifications import mail, mailgun, try_context
 
-logger = logging.getLogger("release")
+logging.basicConfig(level=logging.INFO)
```
Comment (git-harry) [inline scripts/release.py]: This appears to break the debug logging due to the ordering of `logging.basicConfig` calls between it and ghutils, I believe it only has an effect the first time it is called.

File: scripts/release.py [hunk 2]
```diff
@@ -10,7 +10,8 @@
...
+logger = logging.getLogger()
```
Comment (git-harry) [inline scripts/release.py]: Removing the name gives an unhelpful `root` in the log line, `release` is much more descriptive.

**Pull request closed without merging**